runners-manager/vm_creation/github_actions_api.py

```diff
@@ -6,54 +6,49 @@
 
 HEADERS = {
     'Accept': 'application/vnd.github.v3+json',
-    'Authorization': f'token {os.getenv("GITHUB_TOKEN")}'
+    'Authorization': f'token {os.getenv("githubToken")}'
```

Comment:
I made a mistake expressing myself about the camelcase. I meant that it should be only on the helm chart, no need of doing it in the actual code.

Because we usually see upper case when using env variable, and that is probably the direction we are going to move forward with.